frontend/src/views/CollabPageView.tsx

```diff
@@ -50,25 +54,53 @@ const customQuestion: Question = {
 	],
 };
 
-function CollabPageView() {
+const CollabPageView: React.FC = () => {
 	const [code, setCode] = useState("");
-	const [socket, setSocket] = useState<Socket | null>(null);
+   	const [socket, setSocket] = useState<Socket | null>(null);
+   	const [message, setMessage] = useState(""); // For new message input
+   	const [messages, setMessages] = useState<{ username: string; message: string }[]>([]);
+   	const [userId, setUserId] = useState<string>("");
+   	const navigate = useNavigate();
+   	const [questionData, setQuestionData] = useState<Question>(customQuestion);
+   	const { sessionId: sessionIdObj } = useParams<{ sessionId: string }>();
 	const editorRef = useRef<monaco.editor.IStandaloneCodeEditor | null>(null);
 
 	useEffect(() => {
 		// Initialize the WebSocket connection when the component mounts
 		const newSocket = io("http://localhost:5004"); // Backend WebSocket server URL
```

Comment:
We could use the `VITE_COLLAB_SERVICE_BACKEND_URL` here instead of hardcoding the url. 

Though there would be no difference until we deploy publicly, so it's fine if this remains unchanged for now. 😄

Reply:
ok will nto change it for now!